paasta_tools/check_chronos_jobs.py

```diff
@@ -200,13 +199,17 @@ def main():
             except utils.NoDeploymentsAvailable:
                 print(utils.PaastaColors.cyan("Skipping %s because no deployments are available" % service))
                 continue
-            sensu_output, sensu_status = sensu_message_status_for_jobs(
-                chronos_job_config=chronos_job_config,
-                service=service,
-                instance=instance,
-                cluster=cluster,
-                job_state_pairs=job_state_pairs
-            )
+            try:
+                sensu_output, sensu_status = sensu_message_status_for_jobs(
+                    chronos_job_config=chronos_job_config,
+                    service=service,
+                    instance=instance,
+                    cluster=cluster,
+                    job_state_pairs=job_state_pairs
+                )
+            except ValueError:
```

Comment:
Would you be less-un-happy if it used "None" as the signal for "Nothing to send to sensu" instead of an exception?